common/src/main/java/org/apache/rocketmq/common/constant/PermName.java

```diff
@@ -68,4 +68,8 @@ public static boolean isValid(final int perm) {
     public static boolean isPriority(final int perm) {
         return (perm & PERM_PRIORITY) == PERM_PRIORITY;
     }
+
```

Comment:
how about change method name to isInAccessable()? 
and use !isReadable(xxx) && !isWriteable(xxx) to mark this.